src/jit/inlinepolicy.cpp

```diff
@@ -101,8 +115,53 @@ void LegacyPolicy::noteFatal(InlineObservation obs)
 
 void LegacyPolicy::noteInt(InlineObservation obs, int value)
 {
-    (void) value;
-    note(obs);
+    switch (obs) {
```

Comment:
Nit: the opening brace should be on the following line and the extra newline should be removed.